bottomless-cli/Cargo.toml

```diff
@@ -4,7 +4,7 @@ version = "0.1.14"
 edition = "2021"
 license = "MIT"
 keywords = ["libsql", "sqlite", "s3", "cli", "replication"]
-repository = "https://github.com/libsql/sqld"
+repository = "https://github.com/libsql/libsql"
```

Comment:
Well get this fixed eventually as well as the org is wrong now too, but well do that.

Reply:
fixed while i was waiting, heh